validation/jinja2/validation/advanced_search.html

```diff
@@ -0,0 +1,56 @@
+{#
+    # Displays search form to perform an advanced search
+#}
+   
+{% extends 'validation/_base.html' %}
+{% import 'validation/_macros.html' as macros %}
+
+{% block content %}
+<script>
+    function getSelectedSpeakers() {
+        var items = document.getElementsByName('speaker-options')
+        var selected_speakers = "";
+        for (var i=0; i<items.length; i++){
+            if(items[i].type=='checkbox' && items[i].checked==true)
+                selected_speakers += items[i].value+",";
+        }
```

Comment:
Suggest using `for..of` loop:

```suggestion
        for (let speaker of items){
            if (speaker.type === 'checkbox' && speaker.checked)
                selectedSpeakers.push(speaker.value);
        }
```